### ISSUE TITLE
Enable CentOS CI and Deploy latest Rook + Ceph Cluster versions for E2E tests

### ISSUE THREAD

**Issue by humblec** [ceph/ceph-csi#1276]
# Describe the bug #

Working with minikube driver in travis CI has limitations and blocking features to get E2E tested due to the limtation at times. One very recent example would be incapability of bringing up or deploying recent Rook Versions ( >=1.3) thus the E2E tests are running with very old Ceph Cluster versions in place. 

Please refer #394 for more details on the requirement to bring up CentOS CI and the need to overcome this.
This issue track enabling `CentOS CI with latest Rook which allows the E2E cluster to consume latest Ceph Cluster for unblocking new features.

**Comment by Madhu-1** [comment]
https://github.com/ceph/ceph-csi/issues/857 @nixpanic  is already working on which track it

**Comment by humblec** [comment]
sure.. but this also track latest Rook + Ceph Cluster deployment for E2E.  :+1:

**Comment by Madhu-1** [comment]
This is already done.closing this one
